nodejs20.x/hello-img/{{cookiecutter.project_name}}/hello-world/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM public.ecr.aws/lambda/nodejs:20-preview
```

Comment:
I think we need to remove the `preview` tag?

Reply:
```suggestion
FROM public.ecr.aws/lambda/nodejs:20
```